lib/fform/fform.flow

```diff
@@ -152,6 +152,8 @@ export {
 			FVideoRealSize(size : DynamicBehaviour<WidthHeight>);
 			// get current frame as base64 string
 			FVideoGetCurrentFrame(fn : ref () -> string);
+			// to switch files (samples) in multiplayer
+			IndexInPlaylist(index : DynamicBehaviour<int>);
```

Comment:
Missed this at first glance, IndexInPlaylist should go to different union, it means nothing in FVideo case.
Add something like MMultiAudioPlayerStyle ::= FVideoStyle, IndexInPlaylist to material_ui.flow, move IndexInPlaylist there and change mstyle : [FVideoStyle] to mstyle : [MMultiAudioPlayerStyle] in MMultiAudioPlayer.
In case you need to pass mstyle to TVideo, videoPlayerStyle2FVideoStyle in material_player.flow is an example of extracting FVideoStyle from other union.

Reply:
Well, I got it.